src/LFAToolkit.jl

```diff
@@ -33,7 +33,11 @@ export TensorBasis,
     TensorH1LagrangePProlongationBasis,
     TensorHProlongationMacroBasisFrom1D,
     TensorH1LagrangeHProlongationMacroBasis,
-    TensorH1UniformHProlongationMacroBasis
+    TensorH1UniformHProlongationMacroBasis,
+    transformquadrature,
+    sausage_transformation,
+    kosloff_talezer_transformation,
+    hale_trefethen_strip_transformation
 # operato oields
```

Comment:
strange typo here (my fault in an old commit) on line 41 - Should be `operator fields`